packages/wdio-utils/src/index.js

```diff
@@ -5,7 +5,7 @@ import { initialiseWorkerService, initialiseLauncherService } from './initialise
 import webdriverMonad from './monad'
 import {
     commandCallStructure, isValidParameter, getArgumentType, safeRequire,
-    isFunctionAsync
+    isFunctionAsync, transformCommandLogResult
```

Comment:
This might be nitpicking, but why do we have the imports on 2 lines and the imports on line 11 on multiple lines?

Reply:
I think GitHub made this confusing, if you expand the code the imports look similar:

```js
import {
    commandCallStructure, isValidParameter, getArgumentType, safeRequire,
	isFunctionAsync, transformCommandLogResult
}
```
vs.
```js
import {
    wrapCommand, runFnInFiberContext, executeHooksWithArgs,
    hasWdioSyncSupport, executeSync, executeAsync,
} from './shim'
```

In general I would be fine with including an Eslint rule to enforce a similar coding style.